docs/CucumberSyntax.md

```diff
@@ -104,13 +104,13 @@ And there is a constraint:
 
 ### Validating the data in the output
 
-#### Temporal
-* _{field} contains temporal data_, executes the generator and asserts that _field_ contains either `null` or temporal values
-* _{field} contains anything but temporal data_, executes the generator and asserts that _field_ contains either `null` or data that is not temporal.
-* _{field} contains temporal values between {min} and {max} inclusively_, executes the generator and asserts that _field_ contains either `null` or temporal values between _{min}_ and _{max}_. Does so in an inclusive manner for both min and max.
-* _{field} contains temporal values outside {min} and {max}_, executes the generator and asserts that _field_ contains either `null` or temporal values outside _{min}_ and _{max}_.
-* _{field} contains temporal values before or at {before}_, executes the generator and asserts that _field_ contains either `null` or temporal values at or before _{before}_
-* _{field} contains temporal values after or at {after}_, executes the generator and asserts that _field_ contains either `null` or temporal values at or after _{after}_
+#### Datetime
+* _{field} contains datetime data_, executes the generator and asserts that _field_ contains either `null` or datetime values
```

Comment:
I would leave this section referring to "temporal" values, they are temporal, just requested via the "datetime" type. 

In future these steps can be used to verify dates and times independently - if these types are developed, this section would then not need to be updated.